semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Settings.scala

```diff
@@ -1,31 +1,56 @@
 package scala.meta.internal.metacp
 
+import java.io.PrintStream
+
+import org.langmeta.io.AbsolutePath
+import org.langmeta.io.Classpath
+
+import scala.meta.cli.Metacp
+
 final case class Settings(
-    cps: List[String] = Nil,
-    d: String = "."
-)
+    out: PrintStream = System.out,
+    err: PrintStream = System.err,
```

Comment:
If we need an abstraction for `out` and `err`, I suggest we define a `Reporter` that lives outside `Settings`. 

Things like `Settings.parse(args, out, err)` look weird enough to suggest that reporting is a concern that is unrelated to settings.

Reply:
Implemented in https://github.com/scalameta/scalameta/pull/1394.